x-pack/plugins/lists/common/schemas/types/entries.ts

```diff
@@ -52,7 +52,7 @@ export type EntryExists = t.TypeOf<typeof entriesExists>;
 
 export const entriesNested = t.exact(
   t.type({
-    entries: t.array(t.union([entriesMatch, entriesMatchAny, entriesList, entriesExists])),
+    entries: t.array(entriesMatch),
```

Comment:
NOTE: Looking at KQL nested support, looks like it would only support our `match` and not any of the other operators. Please feel free to comment if that's wrong.